package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "prepublish": "npm run build",
     "build": "del-cli dist && cross-env BABEL_ENV=publish babel src --out-dir dist --ignore src/__tests__/**/*.js && jison src/parser.jison -o dist/parser.js",
-    "pretest": "eslint src && npm run build",
+    "pretest": "npm run build && eslint src",
```

Comment:
Eslint report about `Can't resolve `dist/parser`` on first run because we don't have parser before (we create this file in `build` script)